tests/self_peertest.rs

```diff
@@ -206,13 +206,22 @@ async fn peertest_state_offer_contract_bytecode() {
 #[tokio::test(flavor = "multi_thread")]
 #[serial]
 async fn peertest_state_recursive_gossip() {
-    let (peertest, target, handle) = setup_peertest().await;
+    let (peertest, target, handle) = setup_peertest(None).await;
```

Comment:
I think it would be a bit better to get rid of the `Option` and just use `"mainnet"` as the arg here